qmp.opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "cmdliner"
   "ounit2" {with-test}
 ]
-dev-repo: "git://github.com/xapi-project/ocaml-qmp"
+dev-repo: "git+https://github.com/xapi-project/ocaml-qmp"
```

Comment:
I'd use `https://github.com/xapi-project/ocaml-qmp.git`. This works well for cloning.

Reply:
It's used for pushing tags by dune-release, so it can't be just https.